qiskit/quantum_info/operators/measures.py

```diff
@@ -344,7 +344,7 @@ def cvx_bmat(mat_r, mat_i):
     # Objective function
     obj = cvxpy.Maximize(cvxpy.trace(choi_rt_r @ x_r) + cvxpy.trace(choi_rt_i @ x_i))
     prob = cvxpy.Problem(obj, cons)
-    sol = prob.solve(**kwargs)
+    sol = prob.solve(solver="SCS", **kwargs)
```

Comment:
If this works, we should specify the `solver` as a direct `kwarg` in the `diamond_norm` function with a default of `"SCS"` so that users can continue to override it.